text/0000-compact-slates.md

```diff
@@ -0,0 +1,500 @@
+   * `I1` - Invoice (payee to payer) transaction flow, new transaction that incorporates the Invoicer's output(s). Invoicer's signature nonce and excess (with kernel offset) is included and ready for sending to Invoicee.
+   * `I2` - Invoice workflow, Invoicee has added their inputs and change output(s).  Invoicee's signature, nonce and excess is included and ready for sending back to the invoicer.
+   * `I3` - Invoice workflow, Slate complete, contains all inputs, outputs and final signatures, and is ready for posting
+
+#### Signature Data
+
+An entry in the `sigs` array is as follows:
+
+```
+{
+   "xs": "A5sKZUTyGG1FogzBtH8ZvGaVVOFFxusytVLN0rdpX2DE",
+   "part": null,
+   "nonce": "A1MbUjB5g4fjZ3fTYATVBA8TADHGG77D8gHpzbQTl/IE"
+}
+```
+
+The `sigs` struct contains is comprised of an array of participant signature data, with each entry comprising:
+   * `xs` - Base64 encoded short form public key on the secp256k1 curve representing the public blind excess for the participants outputs subtracted from each party's part of the offset.
+   * `part` - Base64 encoded Aggregated (Schnorr) secp2561k signature represeting the participant's partial sig. May be omitted if the participant does not yet have enough data to create it
+   * `nonce` - Base64 encoded The public key of the nonce chosen by the participant for their partial signature
+
+The other party's `sig` entry is removed from the slate before sending it back to the transaction initiator during the S2 and I2 phases.
+
+#### Payment Proof Data
+
+If included, the proof structure is:
+
+```
+  "proof": {
+    "saddr": "7e008eb593ba17d116e282d6267a3c6aad87b910933ad34dfa4d7d2c92b6ba31",
+//  "rsig": null,
+    "raddr": "3a425bd5da5f0f78593251ede7fad0ecf7a95679d84b2cb405255d97ce068234"
+  }
+```
+
+The `proof` struct is an optional payment proof request that must be filled out by the recipient if requested (only valid for basic transaction flow). Its fields are as follows:
+
+   * `saddr` - Base64 encoded short-form public key on the ed25519 curve, representing the sender's wallet address (see the [payment proofs rfc](https://github.com/mimblewimble/grin-rfcs/blob/master/text/0006-payment-proofs.md) for details.
+   * `raddr` - Base64 encoded short-form public key on the ed25519 curve represnting the recipient's wallet address
+   * `rsig` - Base64 encoded EdDSA ed25519 signature representing the recipient's payment-proof signature. Can be omitted if this has not yet been filled out
+
+#### Transaction Object Fields
+
+The V4 Version of the Slate defines its own representation of Grin's internal Tranasction object, which defines its own serialization at [Transaction](https://github.com/mimblewimble/grin/blob/34ff103bb02bc093fe73d36641eb193f7ef2404f/core/src/core/transaction.rs#L871). Internally, the wallet will transform its VersionedSlate into the format expected by the Grin node, allowing the wallet to specify its own Transaction format.
```

Comment:
`The V4 Version of the Slate defines its own representation of Grin's internal Tranasction object, which defines its own serialization at Transaction.`
Is it possible to simplify that sentence? Otherwise okay just hard to understand.